packager/media/formats/webm/webm_cluster_parser.cc

```diff
@@ -259,7 +259,7 @@ bool WebMClusterParser::OnBinary(int id, const uint8_t* data, int size) {
       return true;
 
     case kWebMIdBlockAdditional: {
-      uint64_t block_add_id = base::HostToNet64(block_add_id_);
+      uint64_t block_add_id = absl::big_endian::FromHost16(block_add_id_);
```

Comment:
Looks like this 16 should have been 64.

Reply:
Fixed.  If the tests pass, I'll merge it.

Reply:
Thanks for fixing it!